analyzer/src/functTest/resources/dynamic-project.with-locks-project.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
I couldn't find anything corresponding to the project artifact(s) in the original file which doesn't seem right, so I made one up.

Reply:
I think because that test was verifying dynamic dependencies and lock files so the project version didn't matter hugely.